DataFormats/GEMDigi/interface/GEMPadDigiCluster.h

```diff
@@ -22,7 +22,8 @@ class GEMPadDigiCluster {
 
   explicit GEMPadDigiCluster(std::vector<uint16_t> pads,
                              int16_t bx,
-                             enum GEMSubDetId::Station station = GEMSubDetId::Station::GE11);
+                             enum GEMSubDetId::Station station = GEMSubDetId::Station::GE11,
+                             uint8_t nPart = 8);
```

Comment:
instead of hardcoded 8, use the enum defined above

Reply:
done